Makefile

```diff
@@ -239,7 +239,7 @@ generate-manifests-api: controller-gen ## Generate ClusterRole and CustomResourc
 
 .PHONY: generate-exp-etcdrestore-manifests-api
 generate-exp-etcdrestore-manifests-api: controller-gen ## Generate ClusterRole and CustomResourceDefinition objects for experimental API.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd paths="./exp/etcdrestore/api/..." \
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd paths="./exp/etcdrestore/api/v1alpha1/..." \
```

Comment:
There is a PR which addresses similar problem - https://github.com/rancher/turtles/pull/723

Reply:
should we still keep this Makefile change?

Reply:
We can, but it might be we will not get CRDs generated and installed in a test environment like `envtest` once the functionality is testable